### PR TITLE
pylint added in pre-commit

### PR DESCRIPTION
**Pull request by smriad**

### CODE REVIEW

**Comment by jbampton** [comment]
You will need to test locally before submitting a pr.  Clearly `pre-commit` failed for `pylint`

**Review by jbampton — COMMENTED** [review]
pre-commit is failing

**Comment by BaseMax** [comment]
you need to run `pre-commit`, so it will fix all problems.
it's easy.

File: pylintrc [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+persistent=yes
+
+# List of plugins (as comma separated values of python modules names) to load,
+# usually to register additional checkers.
+load-plugins=
+
+
+[MESSAGES CONTROL]
+#enable=
+disable=E0401, C0103, C0301, E0015, R0903, C0114, E0401, R0201
```
Comment (jbampton) [inline pylintrc:16]: Can you put these in order ?

**Pull request merged**